packages/react/src/components/Button/index.scss

```diff
@@ -4,7 +4,9 @@
   border: thin solid transparent;
   box-sizing: border-box;
   cursor: pointer;
-  display: flex;
+  display: inline-flex;
```

Comment:
Are we sure it does not break anything about the current implementation button ?

Reply:
On the storybook side the form examples looks fine, it tried thinking of breaking use cases and couldn't find .. but maybe I'm wrong. My thinking path was that a button is `inline-block` by default so having it `inline-flex` should not break anything .. but as those positioning are weird it is not bullet proof